fakewallet/src/main/java/com/solanamobile/fakewallet/ui/MainViewModel.kt

```diff
@@ -44,6 +45,14 @@ class MainViewModel(
         viewModelScope.launch {
             observeSeedVaultContentChanges()
             refreshUiState()
+
+            // Privileged wallets do not manually authorize individual seeds, so check on startup
```

Comment:
This fixes privileged wallets not having any accounts marked as user accounts, and thus an incomplete FakeWallet UI